WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -56,6 +56,7 @@
             <% end %>
             <li class="divider"></li>
             <li><a href="/results/statistics.php"><%= icon('area-chart') %> <%= t '.statistics' %></a></li>
+            <li><a href="<%= relations_path %>"><%= icon('sitemap') %> <%= t '.relations' %></a></li>
```

Comment:
I agree with @Laura-O . This is definitely a another subpoint of Statistics, just like Evolution of Records, Sum of Ranks, and Missing Averages.

On another node: Since you guys are already working on things like this, maybe it would be time to port the whole statistics page to Rails? ;)